vendor/wp-now/src/get-wp-cli-path.ts

```diff
@@ -1,13 +1,13 @@
 import path from 'path';
+import getWpNowPath from './get-wp-now-path';
 import getWpCliTmpPath from './get-wp-cli-tmp-path';
-import { getResourcesPath } from '../../../src/storage/paths';
 
 /**
  * The path for wp-cli phar file within the WP Now folder.
  */
 export default function getWpCliPath() {
 	if (process.env.NODE_ENV !== 'test') {
-	  return path.join( getResourcesPath(), 'wp-files', 'wp-cli', 'wp-cli.phar' );
+		return path.join(getWpNowPath(), 'wp-cli.phar');
```

Comment:
Should it be `getServerFilesPath()`?

Reply:
I'm not 100% sure about that. This was initially like that and was changed in https://github.com/Automattic/dotcom-forge/issues/7320.

`getWpNowPath` checks for `STUDIO_SITE_SERVER_PROCESS ` and if this is true, then returns `path.join( appDataPath, process.env.STUDIO_APP_NAME, 'server-files' )` instead of `getServerFilesPath`.  Otherwise it returns `getServerFilesPath`. I guess this was introduced for a reason.
cc: @sejas .

Reply:
Updated!